src/app/project/add.tpl.html

```diff
@@ -39,6 +39,7 @@
                        ng-model="vm.project_name"
                        ng-model-options="{ debounce: 500 }"
                        project-name-available-validator
+                       organization-id="vm.currentOrganization.id"
```

Comment:
I've added support as mentioned in the comment for the `__newOrganization` part in 50de8d7.